packages/app-desktop/gui/Sidebar/styles/index.ts

```diff
@@ -84,26 +84,17 @@ export const StyledListItemAnchor = styled.a`
 	align-items: center;
 	user-select: none;
 	height: 100%;
+
+	// Focus is already shown with a different background
+	&:focus-visible {
+		outline: none;
+	}
```

Comment:
```suggestion
```

While this does fix an issue in the sidebar, this change isn't relevant to the pull request and can safely be moved to a separate pull request.